src/library_nodefs.js

```diff
@@ -1,5 +1,5 @@
 mergeInto(LibraryManager.library, {
-  $NODEFS__deps: ['$FS', '$PATH'],
+  $NODEFS__deps: ['$PATH'],
```

Comment:
Yeah, moving those postsets might be a little confusing. They would be inside `#if __EMSCRIPTEN_HAS_nodefs_js__` in the other file, so at least it's clear to what they belong. But still, I agree it's not optimal. Perhaps there is a third option that is better.

One other possible option is to stop erroring on circular postsets and just say that is undefined behavior, as mentioned before. Perhaps we could add a warning on it in VERBOSE mode. But given it would always warn about FS stuff, that seems not great.